### DISCUSSION TITLE
Welcome to crDroid_gsi Discussions!

### DISCUSSION THREAD

**Discussion by naz664** [naz664/crDroid_gsi#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Disel1977** [comment]
crDroid-10.13 v2025.03.08 a64
Does not start on Samsung SM t-515.
crDroid-10.0-a64_bvN-Unofficial Flashes and starts without problems, but it does not have Gapps, and I really need them.

What are the options for installing this firmware on the t-515 Gapps.

**Comment by cromogamo** [comment]
The latest GSI version with Google apps works great for me, but Bluetooth doesn't work. I only need it for headphones, and for now I can use a cable, but is there a way to change the gesture to one like the one used by Lineage OS in the form of an APK? For now I can use a cable, but solving this would save me a lot of hassle. My phone is a Redmi 10 2022.

**Reply by Disel1977** [reply]
Get this build for the Samsung SM T-515. It works perfectly for me, except for MTP. 
[https://cloud.mail.ru/public/popa/b5FmctyCa](https://4pda.to/stat/go?u=https%3A%2F%2Fcloud.mail.ru%2Fpublic%2Fpopa%2Fb5FmctyCa&e=127944078)
